frigate/models.py

```diff
@@ -19,6 +19,7 @@ class Event(Model):
     box = JSONField()
     area = IntegerField()
     retain_indefinitely = BooleanField(default=False)
+    ratio = FloatField()
```

Comment:
Now that I am on my computer, just leaving an official comment about the migration being needed.

Reply:
@NickM-27 I added the migration script [here](https://github.com/blakeblackshear/frigate/pull/2952/files/2ad0ed9b8d74475e05c800eecd82bf643db580b5#diff-2546ea6e97f5943ff2ab27f9cfd917da1af7eb19d6cdafd6ca96236bf82b2438), are you saying that any default values should match both there and here?

Reply:
Sorry didn't see it, a couple notes:
1. yes the default values should be added here as well
2. There is already a 007 migration so this should be updated to 008